spaceship/lib/spaceship/tunes/tunes_client.rb

```diff
@@ -1014,6 +1027,22 @@ def send_app_submission(app_id, version, data)
       parse_response(r, 'data')
     end
 
+    #####################################################
+    # @!group Reject App Submission
+    #####################################################
+    def reject_app_submission(app_id, version)
```

Comment:
@RishabhTayal This method is still a duplicate of `reject!` https://github.com/RishabhTayal/fastlane/blob/f8b922f7848f2981a742bbad8dd611f1fbf7ee66/spaceship/lib/spaceship/tunes/tunes_client.rb#L1316-L1328